settings_dialog.py

```diff
@@ -72,8 +72,11 @@ def saveState(self):
         Store the options into the user's stored session info.
         """
         mySettings = QtCore.QSettings()
-        mySettings.setValue('svir/platform_hostname',
-                            self.ui.hostnameEdit.text())
+        # if the hostname ends with '/', remove it
+        hostname = self.ui.hostnameEdit.text().strip()
+        if hostname.endswith('/'):
+            hostname = hostname[:-1]
```

Comment:
You can also use .rstrip:

>>> 'pippo/'.rstrip('/')
'pippo'

This is better because it strips any number of slashes:

>>> 'pippo//'.rstrip('/')
'pippo'